src/acktest/bootstrapping/__init__.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import annotations
+
+import abc
+import pickle
+import logging
+
+from pathlib import Path
+from dataclasses import dataclass, fields
+from typing import Iterator
+
+@dataclass
+class ServiceBootstrapResources(abc.ABC):
```

Comment:
how about we call this just `Serializable`, make it a non-abstract base class with only the `serialize` and `deserialize` methods on it.